Source/WebCore/rendering/RenderObject.cpp

```diff
@@ -2553,6 +2553,16 @@ String RenderObject::debugDescription() const
     return builder.toString();
 }
 
+bool RenderObject::isSkippedContent() const
+{
+    return style().isInSkippedContentSubtree() && parent() && parent()->style().isInSkippedContentSubtree();
```

Comment:
I think it sufficient to test the parent bit only.

Reply:
Nice thanks!